### PR TITLE
Fix:#1921 Add CHANGELOG release notes for 1.13

### PR DESCRIPTION
**Pull request by srirambv**
## Summary of Changes

This pull request fixes issue #1921 

## Submitter Checklist:

- [x] *Unit Tests* are updated to cover new or changed functionality
- [x] User-facing strings use `NSLocalizableString()`

## Test Plan:

N/A

## Reviewer Checklist:

- [x] Issues include necessary QA labels:
  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
- [x] Necessary [security reviews](https://github.com/brave/security/issues/new/choose) have taken place.
- [x] Adequate unit test coverage exists to prevent regressions.
- [x] Adequate test plan exists for QA to validate (if applicable).
- [x] Issue is assigned to a milestone (should happen at merge time).

### CODE REVIEW

**Comment by jhreis** [comment]
Waiting on @kjozwiak

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## [1.13](https://github.com/brave/brave-ios/releases/tag/v1.13)
+ 
+ - Implement Brave Rewards and Ads. ([#1920](https://github.com/brave/brave-ios/issues/1920))
```
Comment (kjozwiak) [inline CHANGELOG.md]: Change `Implement` --> `Added` (lets keep it consistent with desktop & iOS)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,21 @@
...
+ - Added new preview menu on iOS13. ([#1499](https://github.com/brave/brave-ios/issues/1499))
+ - Added bookmarklet support. ([#1119](https://github.com/brave/brave-ios/issues/1119))
+ - Updated text size and colours on new tab stats to match desktop. ([#1641](https://github.com/brave/brave-ios/issues/1641))
```
Comment (kjozwiak) [inline CHANGELOG.md]: Change `colours` --> `color` (was asked to use color for desktop so we should keep things consistent)

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,21 @@
...
+ - Updated StartPage icon. ([#1598](https://github.com/brave/brave-ios/issues/1598))
```
Comment (kjozwiak) [inline CHANGELOG.md:9]: nit: not everyone knows that `StartPage` is a search engine. We should add more context. Example:

* Updated StartPage icon under search engine list.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,21 @@
...
+ - Fixed wrong device name being shown when tapped on sync devices. ([#1587](https://github.com/brave/brave-ios/issues/1587))
```
Comment (kjozwiak) [inline CHANGELOG.md]: Change `tapped` --> `tapping`

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,21 @@
...
+ - Fixed readablity of website/tab names on dark mode. ([#1551](https://github.com/brave/brave-ios/issues/1551))
```
Comment (kjozwiak) [inline CHANGELOG.md]: Change `readablity` --> `readability`. Should probably change `on` --> `in`.  Either of these would work:

* Fixed readability of websites/tab names in dark mode.
* Fixed readability of websites/tab names when using dark mode.

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,21 @@
...
+ - Fixed button border on Shields that were too light on light theme. ([#1548](https://github.com/brave/brave-ios/issues/1548))
+ - Fixed missing long press context menu on image search results. ([#1547](https://github.com/brave/brave-ios/issues/1547))
+ - Fixed "Download Cancelled" visibility on iOS 12 devices. ([#1516](https://github.com/brave/brave-ios/issues/1516))
```
Comment (kjozwiak) [inline CHANGELOG.md]: Following sounds a bit better/clearer of what the issue was:

*  Fixed "Download Cancelled" not visible after cancelling a download under iOS 12.

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,21 @@
...
+## [1.13](https://github.com/brave/brave-ios/releases/tag/v1.13)
+ 
+ - Implement Brave Rewards and Ads. ([#1920](https://github.com/brave/brave-ios/issues/1920))
+ - Added new preview menu on iOS13. ([#1499](https://github.com/brave/brave-ios/issues/1499))
+ - Added bookmarklet support. ([#1119](https://github.com/brave/brave-ios/issues/1119))
+ - Updated text size and colours on new tab stats to match desktop. ([#1641](https://github.com/brave/brave-ios/issues/1641))
+ - Updated StartPage icon. ([#1598](https://github.com/brave/brave-ios/issues/1598))
+ - Fixed wrong device name being shown when tapped on sync devices. ([#1587](https://github.com/brave/brave-ios/issues/1587))
+ - Fixed readablity of website/tab names on dark mode. ([#1551](https://github.com/brave/brave-ios/issues/1551))
+ - Fixed button border on Shields that were too light on light theme. ([#1548](https://github.com/brave/brave-ios/issues/1548))
```
Comment (kjozwiak) [inline CHANGELOG.md]: Following sounds a bit better/clearer and conveys that the original issue was:

* Fixed button border visibility under Brave Shields when using the light theme.

**Review by kjozwiak — CHANGES_REQUESTED** [review]

**Comment by kjozwiak** [comment]
@srirambv mind taking a look when you have some time so we can push these to https://brave.com/latest/?

File: CHANGELOG.md (hunk 3 above)
Comment (srirambv) [reply CHANGELOG.md:9]: Startpage shows up as a quick search so dont think if it needs to be explicitly called out.

**Comment by srirambv** [comment]
Fixed. Please review and merge. Didnt change StartPage as its been there in the product already for a long time and users would know about it.

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,21 @@
...
+ - Added Brave Rewards and Ads. ([#1920](https://github.com/brave/brave-ios/issues/1920))
+ - Added new preview menu on iOS13. ([#1499](https://github.com/brave/brave-ios/issues/1499))
+ - Added bookmarklet support. ([#1119](https://github.com/brave/brave-ios/issues/1119))
+ - Updated text size and colors on new tab stats to match desktop. ([#1641](https://github.com/brave/brave-ios/issues/1641))
+ - Updated StartPage icon. ([#1598](https://github.com/brave/brave-ios/issues/1598))
+ - Fixed wrong device name being shown when tapping on sync devices. ([#1587](https://github.com/brave/brave-ios/issues/1587))
+ - Fixed readability of website/tab names in dark mode. ([#1551](https://github.com/brave/brave-ios/issues/1551))
+ - Fixed button border visibility under Brave Shields when using the light theme. ([#1548](https://github.com/brave/brave-ios/issues/1548))
+ - Fixed missing long press context menu on image search results. ([#1547](https://github.com/brave/brave-ios/issues/1547))
+ - Fixed "Download Cancelled" not visible afyter cancelling a download under iOS 12. ([#1516](https://github.com/brave/brave-ios/issues/1516))
```
Comment (kjozwiak) [inline CHANGELOG.md]: Spelling: `afyter` should be `after`

**Review by kjozwiak — CHANGES_REQUESTED** [review]

**Review by kjozwiak — APPROVED** [review]
LGTM @srirambv 👍 Awesome job!

**Pull request merged**